### ISSUE TITLE
Windows Python 3.6 Test Failures

### ISSUE THREAD

**Issue by jakirkham** [numba/numba#2925]
## Reporting a bug

Seeing test failures on Windows with Numba 0.38.0. Snippet of the failures seen on Python 3.6.

```python
======================================================================
FAIL: test_expm1_D_D (numba.tests.test_ufuncs.TestLoopTypesComplexNoPython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1515, in test_template
    self._check_loop(fn, ufunc, loop)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1472, in _check_loop
    self._check_ufunc_with_dtypes(fn, ufunc, letter_types)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1501, in _check_ufunc_with_dtypes
    ulps=ulps)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\support.py", line 257, in assertPreciseEqual
    self.fail("when comparing %s and %s: %s" % (first, second, failure_msg))
AssertionError: when comparing [-0.45969769-0.84147098j         nan       +nanj -0.45969769-0.84147098j
  0.        +0.j         -0.68297786+4.47046238j         nan       +nanj
 -0.68297786+4.47046238j  0.        +0.j        ] and [-0.80123389-0.30955988j         nan       +nanj -0.80123389-0.30955988j
  0.        +0.j         -0.68297786+4.47046238j         nan       +nanj
 -0.68297786+4.47046238j  0.        +0.j        ]: -0.45969769413186023 != -0.801233889653587 within 2.018169754409307e-16 delta : ufunc 'expm1' arrays differ ([array([0. -1.j , nan+nanj, 0. -1.j , 0. +0.j , 1.5+1.5j, nan+nanj,
       1.5+1.5j, 0. +0.j ]), array([-0.80123389-0.30955988j,         nan       +nanj,
       -0.80123389-0.30955988j,  0.        +0.j        ,
       -0.68297786+4.47046238j,         nan       +nanj,
       -0.68297786+4.47046238j,  0.        +0.j        ])]):
args: double
expected [-0.45969769-0.84147098j         nan       +nanj -0.45969769-0.84147098j
  0.        +0.j         -0.68297786+4.47046238j         nan       +nanj
 -0.68297786+4.47046238j  0.        +0.j        ]
got [-0.80123389-0.30955988j         nan       +nanj -0.80123389-0.30955988j
  0.        +0.j         -0.68297786+4.47046238j         nan       +nanj
 -0.68297786+4.47046238j  0.        +0.j        ]
======================================================================
FAIL: test_expm1_F_F (numba.tests.test_ufuncs.TestLoopTypesComplexNoPython)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1515, in test_template
    self._check_loop(fn, ufunc, loop)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1472, in _check_loop
    self._check_ufunc_with_dtypes(fn, ufunc, letter_types)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\test_ufuncs.py", line 1501, in _check_ufunc_with_dtypes
    ulps=ulps)
  File "C:\bld\numba_1525059992876\_test_env\lib\site-packages\numba\tests\support.py", line 257, in assertPreciseEqual
    self.fail("when comparing %s and %s: %s" % (first, second, failure_msg))
AssertionError: when comparing [-0.45969772-0.84147096j         nan       +nanj -0.45969772-0.84147096j
  0.        +0.j         -0.6829779 +4.4704623j          nan       +nanj
 -0.6829779 +4.4704623j   0.        +0.j        ] and [-0.8012339-0.3095599j        nan      +nanj -0.8012339-0.3095599j
  0.       +0.j        -0.6829779+4.4704623j        nan      +nanj
 -0.6829779+4.4704623j  0.       +0.j       ]: -0.45969772 != -0.8012339 within 1.0834966701622761e-07 delta : ufunc 'expm1' arrays differ ([array([0. -1.j , nan+nanj, 0. -1.j , 0. +0.j , 1.5+1.5j, nan+nanj,
       1.5+1.5j, 0. +0.j ], dtype=complex64), array([-0.8012339-0.3095599j,        nan      +nanj,
       -0.8012339-0.3095599j,  0.       +0.j       ,
       -0.6829779+4.4704623j,        nan      +nanj,
       -0.6829779+4.4704623j,  0.       +0.j       ], dtype=complex64)]):
args: single
expected [-0.45969772-0.84147096j         nan       +nanj -0.45969772-0.84147096j
  0.        +0.j         -0.6829779 +4.4704623j          nan       +nanj
 -0.6829779 +4.4704623j   0.        +0.j        ]
got [-0.8012339-0.3095599j        nan      +nanj -0.8012339-0.3095599j
  0.       +0.j        -0.6829779+4.4704623j        nan      +nanj
 -0.6829779+4.4704623j  0.       +0.j       ]
----------------------------------------------------------------------
Ran 6793 tests in 3697.492s
FAILED (failures=7, skipped=75, expected failures=6)
```

xref: https://github.com/conda-forge/numba-feedstock/pull/7
ref: https://ci.appveyor.com/project/conda-forge/numba-feedstock/build/1.0.10/job/tbt52ea0yarvyjk5

<hr>

- [x] I am using the latest released version of Numba (most recent is visible in
 the change log (https://github.com/numba/numba/blob/master/CHANGE_LOG).
- [x] I have included below a minimal working reproducer (if you are unsure how
 to write one see http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports).

**Comment by flutefreak7** [comment]
Should this be looked at again? It's possible that these tests are passing now, but I'm not sure if they've been run against recent versions of numba and python 3.7.

The Windows builds for the numba feedstock on conda-forge are still turned off due to this issue.

**Comment by stuartarchibald** [comment]
Closing this issue as it seems to be resolved (Numba latest is 0.49.x). If this is not the case please re-open with a comment about any item that appears to be unresolved. Many thanks.

**Comment by jakirkham** [comment]
Thanks Stuart! 😄
